Backendless/Persistence/DataQueryBuilder.cs

```diff
@@ -40,6 +42,17 @@ public BackendlessDataQuery Build()
       return dataQuery;
     }
 
+    public DataQueryBuilder SetDistinct( Boolean distinct )

```

Comment:
@holailusoria  why not use .NET get/set here?

Reply:
@markpiller  DataQueryBuilder uses methods to work with fields. Here I decided not to make an exception.
In addition, the SetDistict method allows you to return a DataQueryBuilder object and continue working with it; what can't be done with .net properties